.gitignore

```diff
@@ -1,7 +1,8 @@
 .idea
 node_modules
 npm-debug.log
-public
```

Comment:
We added an images folder to /public for our logo, which doesn't really need to be part of the repository, but seemed the easiest way to sync amongst ourselves.